WorkloadManagementSystem/ConfigTemplate.cfg

```diff
@@ -88,6 +88,7 @@ Services
       Services =
     }
```

Comment:
I think this `Dependencies` part is useless (and confusing). When was introduced here?

Reply:
Just remove it.